backend/handlers/jointrip.js

```diff
@@ -1,112 +0,0 @@
-const trip = require('../../database/tripHelpers');
-const user = require('../../database/userhelpers');
-const utiles = require('../utils.js');
-module.exports = (req, res) => {
-  const usertripinfo = [req.state.sid.user_id, req.payload.trip_id];
-  trip.getusertripbytripisuserid(usertripinfo, (error, res1) => {
-    if (error) {
-      // eslint-disable-next-line no-console
-      console.log('get user trip by trip id user id error :',error)
-      return res().code(500)
-    }
-    if (res1.rows.length > 0) {
-      res({
-        msg: 'User is already in this Trip',
-      }).code(401)
-    } else {
-      trip.getTripByid({trip_id:usertripinfo[1]},(error,result)=>{
-        if (error) {
-          // eslint-disable-next-line no-console
-          console.log('get trip by tripid error :',error)
-          return res().code(500)
-        }
-
-        if ( result.rows[0].user_id==usertripinfo[0])
```

Comment:
is there a better way to represent this value, rather than an index in an array? It's hard to understand what's being referenced with rows[0] and usertripinfo[0]